### DISCUSSION TITLE
Archiving of funds/labs

### DISCUSSION THREAD

**Discussion by wes-otf** [OpenTechFund/opentech.fund#4015] [General]
Staff currently "archive" unused/unneeded fund that they want to keep the submissions for by renaming it to "<Fund> (archive)". This causes for quite a bit of bloat in terms of applications that are inactive showing up in search results, funds in filter menu options, etc. Ideally the switch for including archived submissions in search results would also enable/disable the showing of these in results.

**Comment by frjo** [comment]
Why not archive these submission as well?

**Comment by wes-otf** [comment]
I figured if the fund was ever to be un-archived and all the submissions had to be un-archived as well it would make things a little weird if any of them were previously individually archived. In my mind it made more sense to check if the overarching fund was archived and make it display similarly to the current archive message now but stating it was the fund, not the individual submission. Let me know if I conveyed that properly

**Comment by frjo** [comment]
We could add an archive toggle on funds and labs. Then a submission is considered archived if the submission is archiver or if the fund/lab it belongs to are archived.

A worry is that it will by confusing to figure out why a specific submission is archived. Is it for reason a or reason b? In one instance you unarchive on the submission in another you need to go to the fund/lab.

**Comment by wes-otf** [comment]
yeah that makes sense to me! I think maybe being clear in the archive message would alleviate some of the confusion there, and even giving some direction in terms of a link/button to view the archived fund's settings in wagtail if they had permissions to do so. 
<img width="1419" alt="archive" src="https://github.com/HyphaApp/hypha/assets/145372368/4598a36c-d810-4b00-bd34-15e4f8188106">

Verbiage instead could be something like `The fund associated with this submission has been archived...`. Just spitballing on all of this, if you feel there would be a way to avoid headache better I'm open to anything
